### PR TITLE
Add GQL status to the UNRECOGNIZED category

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-151.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-151.surge.sh/status-codes/5/notifications/all-notifications)

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 1]
```diff
@@ -950,8 +1060,30 @@ m|WARNING
...
+======
+[.include-with-GQLSTATUS-code]
+======
+
+[cols="<1s,<4"]
+|===
+|GQLSTATUS code
+m|01N51
+|StatusDescription
+a|warn: unknown relationship type. The relationship type `$rel` does not exist. Verify that the spelling is correct.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1072]:
```suggestion
a|warn: unknown relationship type. The relationship type `$reltype` does not exist. Verify that the spelling is correct.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 2]
```diff
@@ -985,8 +1144,29 @@ m|WARNING
...
-====
+======
+[.include-with-GQLSTATUS-code]
+======
+[cols="<1s,<4"]
+|===
+|GQLSTATUS code
+m|01N52
+|StatusDescription
+a|warn: unknown property key. The property `$prop` does not exist. Verify that the spelling is correct.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1155]:
```suggestion
a|warn: unknown property key. The property `$propkey` does not exist. Verify that the spelling is correct.
```

**Review by Lojjs — COMMENTED** [review]
Looks good if you include the two parameter name changes you have suggested yourself.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-151.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-151.surge.sh/status-codes/5/notifications/all-notifications)

**Review by Lojjs — APPROVED** [review]

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 3]
```diff
@@ -930,14 +1010,44 @@ Description of the returned code::
...
+01N50
+
+Returned Status Description::
+warn: unknown label.
+The label `Perso` does not exist.
+Verify that the spelling is correct.
+
+Suggestions for improvement::
+Make sure you didn't misspell the label.
+If nodes with the given label will be created in the future, then no change is needed.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1037]:
```suggestion
If you plan to create nodes with that label in the future, no change is needed.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 4]
```diff
@@ -930,14 +1010,44 @@ Description of the returned code::
 One of the labels in your query is not available in the database, make sure you didn't misspell it or that the label is available when you run this statement in your application (the missing label name is: `Perso`)
 
 Suggestions for improvement::
-Make sure you didn't misspell the label. If nodes with the given label will be created in the future, then no change is needed.
+Make sure you didn't misspell the label.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1013]:
```suggestion
Verify that the label is not misspelled.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 5]
```diff
@@ -930,14 +1010,44 @@ Description of the returned code::
...
+If nodes with the given label will be created in the future, then no change is needed.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1014]:
```suggestion
If you plan to create nodes with that label in the future, then no change is needed.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 6]
```diff
@@ -967,12 +1099,39 @@ Suggestions for improvement::
...
+01N51
+
+Returned Status Description::
+warn: unknown relationship type.
+The relationship type `NonExistingType` does not exist.
+Verify that the spelling is correct.
+
+Suggestions for improvement::
+Make sure you did not misspell the relationship type.
+If relationships will be created, with the given relationship type, in the future, then no change to the query is needed.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1122]:
```suggestion
If you plan to create relationships of this type in the future, no change is needed.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 7]
```diff
@@ -967,12 +1099,39 @@ Suggestions for improvement::
...
+Returned GQLSTATUS code::
+01N51
+
+Returned Status Description::
+warn: unknown relationship type.
+The relationship type `NonExistingType` does not exist.
+Verify that the spelling is correct.
+
+Suggestions for improvement::
+Make sure you did not misspell the relationship type.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1121]:
```suggestion
Verify that the relationship type is not misspelled.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 8]
```diff
@@ -967,12 +1099,39 @@ Suggestions for improvement::
 Make sure you did not misspell the relationship type.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1099]:
```suggestion
Verify that the relationship type is not misspelled.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 9]
```diff
@@ -967,12 +1099,39 @@ Suggestions for improvement::
...
 If relationships will be created, with the given relationship type, in the future, then no change to the query is needed.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1100]:
```suggestion
If you plan to create relationships of this type in the future, no change is needed.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 10]
```diff
@@ -999,9 +1179,33 @@ One of the property names in your query is not available in the database,
 make sure you didn't misspell it or that the label is available when you run this statement in your application (the missing property name is: `nonExistingProp`)
 
 Suggestions for improvement::
-Make sure you didn't misspell the property key. If the property key will be created in the future, then no change is needed to the query.
+Make sure you didn't misspell the property key.
+If the property key will be created in the future, then no change is needed to the query.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1183]:
```suggestion
If you plan to create that property key in the future, no change is needed.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 11]
```diff
@@ -999,9 +1179,33 @@ One of the property names in your query is not available in the database,
...
+01N52
+
+Returned Status Description::
+warn: unknown property key.
+The property `nme` does not exist.
+Verify that the spelling is correct.
+
+Suggestions for improvement::
+Make sure you didn't misspell the property key.
+If the property key will be created in the future, then no change is needed to the query.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1206]:
```suggestion
If you plan to create that property key in the future, no change is needed.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 12]
```diff
@@ -999,9 +1179,33 @@ One of the property names in your query is not available in the database,
...
+Returned GQLSTATUS code::
+01N52
+
+Returned Status Description::
+warn: unknown property key.
+The property `nme` does not exist.
+Verify that the spelling is correct.
+
+Suggestions for improvement::
+Make sure you didn't misspell the property key.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1205]:
```suggestion
Verify that the property key is not misspelled.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 13]
```diff
@@ -999,9 +1179,33 @@ One of the property names in your query is not available in the database,
 make sure you didn't misspell it or that the label is available when you run this statement in your application (the missing property name is: `nonExistingProp`)
 
 Suggestions for improvement::
-Make sure you didn't misspell the property key. If the property key will be created in the future, then no change is needed to the query.
+Make sure you didn't misspell the property key.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1182]:
```suggestion
Verify that the property key is not misspelled.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 14]
```diff
@@ -899,12 +928,40 @@ This command will not take effect until this database is created. (HOME DATABASE
 Suggestions for improvement::
 Make sure the home database has not been misspelled.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:929]:
```suggestion
Verify that the home database name is not misspelled.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 15]
```diff
@@ -899,12 +928,40 @@ This command will not take effect until this database is created. (HOME DATABASE
...
+Returned GQLSTATUS code::
+00N50
+
+Returned Status Description::
+note: successful completion - home database not found.
+The database `Ne4j` does not exist.
+Verify that the spelling is correct or create the database for the command to take effect.
+
+Suggestions for improvement::
+Make sure the home database has not been misspelled.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:951]:
```suggestion
Verify that the home database name is not misspelled.
```

**Review by renetapopova — COMMENTED** [review]
Thanks, @NataliaIvakina! I added some suggestions to avoid using Present Perfect and past tenses. Please feel free to improve them or ignore them.

**Comment by NataliaIvakina** [comment]
> Thanks, @NataliaIvakina! I added some suggestions to avoid using Present Perfect and past tenses. Please feel free to improve them or ignore them.

Good suggestions! Thank you

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-151.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-151.surge.sh/status-codes/5/notifications/all-notifications)

**Pull request merged**